src/components/overview/Overview.jsx

```diff
@@ -126,7 +126,7 @@ const Overview = ({productId}) => {
 
   return (
     <>
-      <div id='announce'><b>&nbsp;FREE SHIPPING&nbsp;+&nbsp;RETURNS,&nbsp; SAVE UP TO 30% ON THE SELECTED ITEMS</b></div>
+      <div id='announce'><b>&nbsp;FREE SHIPPING&nbsp;+&nbsp;RETURNS,&nbsp; SAVE UP TO 30% ON SELECT ITEMS</b></div>
```

Comment:
thank you for changing this!

Reply:
Hope you didn't mind! I missed it when you PR'd it originally